Src/WitsmlExplorer.Frontend/components/ContentViews/EditNumber.tsx

```diff
@@ -0,0 +1,80 @@
+import { Icon, Label, TextField } from "@equinor/eds-core-react";
+import { Tooltip } from "@material-ui/core";
+import { ReactElement, useContext, useState } from "react";
+import styled from "styled-components";
+import OperationContext from "../../contexts/operationContext";
+import { TooltipLayout } from "../ContextMenus/OptionsContextMenu";
+import { StyledButton } from "./EditInterval";
+
+interface EditNumberProps {
+  label: string;
+  infoTooltip?: string;
+  infoIconColor?: string;
+  defaultValue?: number;
+  onSubmit: (value: number) => void;
+}
+
+const EditNumber = (props: EditNumberProps): ReactElement => {
+  const { label, infoTooltip, infoIconColor, defaultValue = 0, onSubmit } = props;
+  const {
+    operationState: { colors }
+  } = useContext(OperationContext);
+  const [isEdited, setIsEdited] = useState(false);
+  const [value, setValue] = useState<string>(String(defaultValue));
+
+  const submitEditNumber = () => {
+    setIsEdited(false);
+    onSubmit(parseFloat(value) || null);
+  };
+
+  const onInputChange = (e: any) => {
```

Comment:
I suggest avoiding the use of the 'any' event type. The [React TypeScript Cheatsheets](https://react-typescript-cheatsheet.netlify.app/) may contain valuable information about available types.

Reply:
True, and this should probably be done elsewhere in the codebase in the future as well. Changed to `ChangeEvent<HTMLInputElement>`